### DISCUSSION TITLE
Where did the Performance Dashboard go?

### DISCUSSION THREAD

**Discussion by HannanSolo** [openthread/openthread#9662] [Q&A] [answered]
I stumbled upon this great [pdf](https://openthread.io/testing/OpenThread_Quality_Dashboard.pdf) from a friend, referencing a performance dashboard that outlines all the testing done with openthread. Yet, the link to the [dashboard](https://openthread.io/testing/performance-dashboard)  seems to redirect me to the testing page with nothing referencing the dashboard.

Any idea where I can find this missing dashboard?

**Comment by jwhui** [comment, marked as answer]
Unfortunately, we did not have the resources to maintain the page, which involves periodically running the tests again, collecting the results, and publishing them on the dashboard. Rather than keeping stale performance data around for a commit several years old, we decided to remove the page from the developer site.

**Reply by HannanSolo** [reply]
Thank you for the answer. Is there anywhere I could see those historical results? I am trying to start a fairly big project and want to know what type of latency I am expected to face with 200+ nodes

**Reply by jwhui** [reply]
Unfortunately, we don't have any archived page to show the historical results.

One resource to check out is Silicon Labs' [Thread Mesh Network Performance](https://www.silabs.com/documents/login/application-notes/an1141-thread-mesh-network-performance.pdf) application note.
